docs/source/model_parallel.rst

```diff
@@ -0,0 +1,11 @@
+.. role:: hidden
+    :class: hidden-section
+
+Distributed RPC Framework - torch.distributed.autograd and torch.distributed.rpc
+=====================================================
+
+.. automodule:: torch.distributed.rpc
+.. currentmodule:: torch.distributed.rpc
+
+.. autofunction:: rpc_sync
```

Comment:
Is there a way to automatically pull in all public methods of a module instead of specifying each one like this?

Reply:
@pritamdamania87 it looks like other doc-related commits list out each public method, see for example https://github.com/pytorch/pytorch/commit/8915e2710c57cefd4bb5a7876216f08d3bc8c92f#diff-7aa5f4b7f9dedcb7a6ce129b9d3c5eb8. The sphix documentation (http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) also makes it seems like this needs to be done - there is support for recursively including members of a class but many of our functions do not belong to a specific class.